src/redux/slices/QuranReader/readingView.ts

```diff
@@ -7,9 +7,7 @@ export const readingViewSlice = createSlice({
   name: 'readingView',
   initialState,
   reducers: {
-    setReadingView: (state: ReadingView, action: PayloadAction<ReadingView>) => {
-      state = action.payload;
-    },
+    setReadingView: (draft, action: PayloadAction) => action.payload,
```

Comment:
Why are we doing this change?

Reply:
you don't need to do `state = `, if you are resetting the whole state, returning the new value is enough.
That will cause the re-assign to fail even with my change in place, as my change is only to allow properties re-assigning, not the entire parameter.